.github/workflows/conformance.yml

```diff
@@ -0,0 +1,41 @@
+name: Conformance Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request_target:
+    types: [labeled]
+  workflow_dispatch:
+
+jobs:
+  conformance:
+    strategy:
+      matrix:
+        java-version: [11, 17]
+      fail-fast: false
+
+    permissions:
+      # Needed to access the workflow's OIDC identity.
+      id-token: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@d171c3b028d844f2bf14e9fdec0c58114451e4bf
+
+      - name: Set up JDK ${{ matrix.java-version }}
+        uses: actions/setup-java@v3
+        with:
+          java-version: ${{ matrix.java-version }}
+          distribution: 'temurin'
+
+      - name: Build sigstore-java
+        uses: gradle/gradle-build-action@3fbe033aaae657f011f88f29be9e65ed26bd29ef # tag=v2.3.3
+        with:
+          arguments: clean :sigstore-conformance:build
```

Comment:
There's no need to have `clean` here

Reply:
[f5f7d14](https://github.com/sigstore/sigstore-java/pull/245/commits/f5f7d140b60bb515f0313db0a1459a2f4e781afe)